tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -366,3 +367,41 @@ def test_positive_virtwho_roles(session):
             assert session.role.search(role_name)[0]['Name'] == role_name
             assigned_permissions = session.filter.read_permissions(role_name)
             assert sorted(assigned_permissions) == sorted(role_filters)
+
+
+@tier2
+def test_positive_virtwho_configs_widget(session, form_data):
+    """Check if Virt-who Configurations Status Widget is working in the Dashboard UI
+
+    :id: 6bed6d55-2bd5-4438-9f72-d48e78566789
+
+    :Steps:
+
+        1. Create a Virt-who Configuration
+        2. Navigate Monitor -> Dashboard
+        3. Review the Virt-who Configurations Status widget
+
+    :expectedresults: The widget is updated with all details.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Low
+    """
+    org = entities.Organization().create()
```

Comment:
Why do you not use the 'Default Organization' ? is it necessary to create a new one?

Reply:
We don't know how much virt-who-configuration and what is the status of them in Default Org when we test this case. In other words, if we use the default Org, we should count the existing virt-who configs, get the status of everyone, and sorted them first, that would be more complicated.

Reply:
Make sense to me, thank you